File: examples-sdk/csharp/602-db-mongo-crud/Main.cs
// 602-db-mongo-crud — C# SDK equivalent
// Compile: vil compile --from csharp --input 602-db-mongo-crud/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("app", 8080);
server.Compile();

File: examples-sdk/java/602-db-mongo-crud/Main.java
// 602-db-mongo-crud — Java SDK equivalent
// Compile: vil compile --from java --input 602-db-mongo-crud/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("app", 8080);
        server.compile();
    }
}
